.npmignore

```diff
@@ -0,0 +1,15 @@
+*.swp
+.ncu
+.nyc_output
+node-gyp-*.tgz
+/.github/
+/docs/
+/gyp/.github/
+/gyp/*.md
+/gyp/AUTHORS
+/gyp/tools/
+/node-gyp-*.tgz
+/test/
+/update-gyp.py
+/gyp/test
+/test/.node-gyp
```

Comment:
I like it when these are sorted to make it easy to spot missing items and nearly impossible to insert duplicates.
https://www.google.com/search?q=sort+lines+in+vscode

Reply:
Sorted both the .npmignore and .gitignore files.